### PR TITLE
Audit and update dependencies

### LINKED ISSUE #250: Audit and update dependencies

### PR DESCRIPTION
**Pull request by danielstreif**
**What**:
Audited and updated dependencies

Fixes #250

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "fb": "^2.0.0",
     "helmet": "^3.21.1",
     "i18n": "^0.8.3",
+    "minimist": "^1.2.5",
```
Comment (pheeria) [inline package.json:52]: I think we don't need this one 🙂 
The problem with audit lies within the dependencies of the dependencies.
So, what needs to be done is `npm outdated` and then look and update outdated dependencies, some of which in their turn rely on `minimist`. 🙂

Comment (danielstreif) [reply package.json]: @pheeria I installed the latest versions of `jest` and `nodemon` and updated other packages to their latest wanted version. Everything seems to work for me and `npm audit` does not detect any vulnerabilities.

**Comment by pheeria** [comment]
Can you please also remove lockfile and `npm install` again?
This would refresh some of the sub-dependencies :)

**Comment by danielstreif** [comment]
@pheeria Good to merge now? I did update the lockfile.

**Comment by pheeria** [comment]
Jawohl!

**Pull request merged**